### PR TITLE
1973 list open contribution links in the wallet

### LINKED ISSUE #1973: 🚀 [Feature] List open contribution links in the wallet
## 🚀 Feature

If contribution links are created in the admin area, this must be visible to all community members for transparent purposes. 
A simple list of current and open links is sufficient. (Name, Description, Start, End, Amount, Cycle, Repeat, MaxAmuont)

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
Übersicht für eine Gemeinschaft
- Community Name
- Community  Beschreibung
- offene Beitraglinksliste
- Moderatorenliste
- Mitgliederzahl

### Issues
 - fixes #1973  

### Merge
-  #2041 
- #2116 
 
image

### CODE REVIEW

**Comment by ogerly** [comment]
@Mogge @Elweyn We could put the statistics from moriz on this page. 
I also need a query that lists the moderators (admins) and the number of members.

**Comment by ogerly** [comment]
i have bookmarked #2041 and have the data displayed. it would be good if this could be seen by all members.

**Comment by ogerly** [comment]
image

**Comment by BerndHueckstaedt** [comment]
Do the contriubution links not belong in the admin interface?

Gehören die Contriubution -Links gehören nicht ins Admin-Interface?

**Comment by ogerly** [comment]
> Do the contriubution links not belong in the admin interface?
> 
> Gehören die Contriubution -Links gehören nicht ins Admin-Interface?

but you can also see them on the overveiw page ... where you can also see them.

File: frontend/src/locales/en.json [hunk 1]
```diff
@@ -224,6 +230,7 @@
   "navigation": {
     "admin_area": "Admin Area",
     "community": "Community",
+    "info": "Info",
```
Comment (Tirokk) [inline frontend/src/locales/en.json:233]:
```suggestion
    "info": "Information",
```

Is much better in my eyes!

File: frontend/src/locales/de.json [hunk 2]
```diff
@@ -224,6 +230,7 @@
   "navigation": {
     "admin_area": "Adminbereich",
     "community": "Gemeinschaft",
+    "info": "Info",
```
Comment (Tirokk) [inline frontend/src/locales/de.json:233]:
```suggestion
    "info": "Information",
```

Is much better in my eyes!

**Review by Tirokk — CHANGES_REQUESTED** [review]
Hey @ogerly 🍀

This is a very important and nice feature you did !!! 🚀🚀💫💫

I have some suggestions:

! Please review the spanish locale @Mogge !

Is the short word `Info` optimal in German and English. I think `Information` is better in both languages.
Even the path should be `/information` I think.

We should have a circle around the "i" as I put in my code suggestions? See screenshot:

Bildschirmfoto 2022-08-18 um 11 56 21

File: frontend/src/locales/es.json [hunk 3]
```diff
@@ -26,8 +26,13 @@
     "community": "Comunidad",
     "continue-to-registration": "Continuar con el registro",
     "current-community": "Comunidad actual",
+    "members": "Miembros",
+    "moderators": "Moderadores",
     "myContributions": "Mis contribuciones al bien común",
+    "openContributionLinks": "lista de enlaces de contribuciones públicas",
+    "openContributionLinkText": "La comunidad {count} proporciona actualmente las siguientes \"{name}\" creaciones automáticas.",
```
Comment (Mogge) [inline frontend/src/locales/es.json:33]:
```suggestion
    "openContributionLinkText": "La comunidad \"{name}\" proporciona actualmente las siguientes {count} creaciones automáticas.",
```

**Review by Mogge — CHANGES_REQUESTED** [review]

File: frontend/src/locales/en.json [hunk 4]
```diff
@@ -26,8 +26,13 @@
     "community": "Community",
     "continue-to-registration": "Continue to registration",
     "current-community": "Current community",
+    "members": "Members",
+    "moderators": "Moderators",
     "myContributions": "My contributions to the common good",
+    "openContributionLinks": "open Contribution links list",
+    "openContributionLinkText": "The following {count} automatic creations are currently provided by the \"{name}\" community.",
```
Comment (Tirokk) [inline frontend/src/locales/en.json:33]:
```suggestion
    "openContributionLinkText": "The following {count} automatic creations are currently provided by the “{name}” community.",
```

**Review by Tirokk — CHANGES_REQUESTED** [review]
Minor suggestion for english quotes …

File: frontend/src/locales/en.json (hunk 4 above)
Comment (ogerly) [reply frontend/src/locales/en.json:33]: modified by hand

**Review by Tirokk — CHANGES_REQUESTED** [review]
Please these little changes again …

**Review by Tirokk — APPROVED** [review]
Yes !!!

I approve 🚀🚀💫💫

**Pull request merged**